src/main/resources/assets/minecolonies/lang/en_US.lang

```diff
@@ -124,6 +124,10 @@ com.minecolonies.gui.townHall.recall=Recall Citizens
 item.scepterSteel.point=Point one saved
 achievement.upgrade.builder.max=The Builder!
 achievement.townhall.desc=Create your own Settlement
+achievement.upgrade.townhall.first=We need a townhall!
```

Comment:
@Raycoms @Kostronor Did these strings get added to POE? or Do we add them on merge?